shopgate/pwa
# Included additional portal props in the SuggestionList component for both themes.

# Description

Included additional portal props in the SuggestionList component for both themes.

## Type of change

Please add an "x" into the option that is relevant:


## How to test it

Inspect the dom in the app and observe the additional portal props to have been added in the portal definitions of the `SuggestionList` component.
